### PR TITLE
feat(ui): Refresh ProGuard settings UI

### PR DESCRIPTION
**Pull request by matejminar**
image

getsentry feature flag: https://github.com/getsentry/getsentry/pull/5094

### CODE REVIEW

**Review by bruno-garcia — APPROVED** [review]
LGTM

File: src/sentry/static/sentry/app/views/settings/projectProguard/projectProguardRow.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        <Name>{debugId || uuid || `(${t('empty')})`}</Name>
+        <TimeWrapper>
+          <IconClock size="sm" />
+          <TimeSince date={dateCreated} />
+        </TimeWrapper>
+      </NameColumn>
+      <SizeColumn>
+        <FileSize bytes={size} />
+      </SizeColumn>
+      <ActionsColumn>
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/settings/projectProguard/projectProguardRow.tsx:43]: should we just use `SizeColumn` here?

**Review by priscilawebdev — APPROVED** [review]
left 1 suggestion but it looks good to me

File: src/sentry/static/sentry/app/views/settings/projectProguard/projectProguardRow.tsx (hunk 1 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/views/settings/projectProguard/projectProguardRow.tsx:43]: We could, I felt like this was more semantic 🤷‍♂️

**Pull request merged**